apps/account/tests/test_views/test_passkey_reg_begin_view.py

```
@@ -0,0 +1,43 @@
+        client = hx_client(user)
+        response = client.get(reverse("account:passkey-reg-begin"))
+
+        assert response.status_code == http.HTTPStatus.BAD_REQUEST
+        data = json.loads(response.content)
+        assert data["status"] == "ERR"
+
+    def test_get_delegates_to_library_when_no_passkey(self, hx_client, user, monkeypatch):
+        fake_state = {"challenge": "abc"}
+        fake_options = {"publicKey": {"challenge": "abc"}}
+
+        def fake_begin_registration(user, request):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `fake_begin_registration`

(ANN202)